core/integration/module_php_test.go

```diff
@@ -18,9 +20,7 @@ func (PHPSuite) TestInit(ctx context.Context, t *testctx.T) {
 	t.Run("from upstream", func(ctx context.Context, t *testctx.T) {
 		c := connect(ctx, t)
 
-		modGen := c.Container().From(golangImage).
-			WithMountedFile(testCLIBinPath, daggerCliFile(t, c)).
-			WithWorkdir("/work").
+		modGen := daggerCliBase(t, c).
 			With(daggerExec("init", "--name=bare", "--sdk=github.com/dagger/dagger/sdk/php"))
```

Comment:
I just made similar changes to the TestInit function. I think this change can be removed.